### DISCUSSION TITLE
config.json file permissions

### DISCUSSION THREAD

**Discussion by utezduyar** [ptz0n/homebridge-verisure#96] [General]
I am trying to increase the security of my installation and I am not happy that credentials are stored in config.json file. Using environment variables can be one option. Another thing I was wondering is that if anyone else but "homebridge" user is reading the json file. If not, then maybe it is safe to remove the read attribute for anyone that is not homebridge user or in homebridge group. 

`$ ls -alh config.json 
-rw-r--r-- 1 homebridge homebridge 522 Dec 18 12:38 config.json`

**Comment by ptz0n** [comment]
Great topic! We should all be cautious about where we store sensitive information. Personally I'm running Homebridge via PM2 and have limited the read rights like you are suggesting to minimise exposure.

Also, I have a separate user setup in Verisure just for IoT. Which makes it dead easy when rotating secrets. Because we all do at a frequent interval, right?
